tests/integration/common/checks-after-conversion/test_enabled_repositories.py

```diff
@@ -28,7 +28,7 @@ def test_enabled_repositories(shell, system_release):
     enabled_repos = shell("yum repolist").output
 
     try:
-        if "redhat-8.6" in system_release:
+        if ("redhat-8.6" or "redhat-8.8") in system_release:
```

Comment:
I don't think this will ever be true for `redhat-8.8` case. The reason for that is that Python will evaluate the first parenthesis and will check if it will use either `redhat-8.6` and `redhat-8.8`, and since both strings exists and are present, it will just default to the first one. 

For example, take a look at this example by changing `system_release` to `redhat-8.8`: 
```bash
Python 3.11.4 (tags/v3.11.4:d2340ef, Jun  7 2023, 05:45:37) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> system_release = "redhat-8.8"
>>> ("redhat-8.6" or "redhat-8.8") in system_release
False
>>> system_release = "redhat-8.6"
>>> ("redhat-8.6" or "redhat-8.8") in system_release
True
```

And look what happens when I executed only the parenthesis part: 

```bash
>>> ("redhat-8.6" or "redhat-8.8")
'redhat-8.6'
```

```suggestion
        if "redhat-8.6" in system_release or "redhat-8.8" in system_release:
```

Reply:
Good catch, I'll add this change thank you